apps/src/applab/applab.js

```diff
@@ -1004,6 +1015,58 @@ Applab.init = function(config) {
   }
 };
 
+Applab.onMouseDownDebugResizeBar = function (event) {
+  // When we see a mouse down in the resize bar, start tracking mouse moves:
+
+  if (event.srcElement.id === 'debugResizeBar') {
```

Comment:
The event listener is added to the debugResizeBar directly. For which srcElements would you not want to trigger the mousedown behavior below?